### ISSUE TITLE
Add colors to whitelist

### ISSUE THREAD

**Issue by chasegiunta** [simplygoodwork/start#14]
This probably shouldn't be a default, but a good thing to have commented to comment out when needed.

This way we're not having to worry about
```class="border-{{ link.active ? 'blue' : 'transparent' }}"```

This is what I'm doing on Freeman but it's total suckage. Since it's a js file we should be able to do something dynamic and baller: importing `config.colors`, looping, generating this array with prefixed & suffixed `/`.

```
    purgecssWhitelistPatterns: [
      /transparent/,
      /black/,
      /grey/,
      /white/,
      /red/,
      /orange/,
      /yellow/,
      /green/,
      /lime/,
      /teal/,
      /blue/,
      /sky/,
      /indigo/,
      /purple/,
      /pink/,
      /purple2/,
      /water/
    ],
```

**Comment by chrisrowe** [comment]
Yeah, I'm flipping back and forwards on whether this is a good default, I think I'd rather see a bit more CSS than stuff breaking more on production because of this issue.
